src/System.Linq/src/System.Linq.csproj

```diff
@@ -23,6 +23,7 @@
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
     <Compile Include="System\Linq\Enumerable.cs" />
+    <Compile Include="System\Linq\Enumerable.Sum.cs" />
```

Comment:
Why the change to separate out just the Sum functions as part of a PR focused on tests?  Seems like if we want to split this file up, it should be a separate discussion/refactoring, at which point we'd do it for all of the members and not just for Sum.